src/test/java/org/jabref/model/groups/TexGroupTest.java

```diff
@@ -28,10 +38,20 @@ public void containsReturnsTrueForEntryInAux() throws Exception {
     @Test
     public void containsReturnsTrueForEntryNotInAux() throws Exception {
         Path auxFile = Paths.get(TexGroupTest.class.getResource("paper.aux").toURI());
-        TexGroup group = new TexGroup("paper", GroupHierarchyType.INDEPENDENT, auxFile, new DefaultAuxParser(new BibDatabase()), new DummyFileUpdateMonitor());
+        TexGroup group = new TexGroup("paper", GroupHierarchyType.INDEPENDENT, auxFile, new DefaultAuxParser(new BibDatabase()), new DummyFileUpdateMonitor(), metaData);
         BibEntry notInAux = new BibEntry();
         notInAux.setCiteKey("NotInAux2017");
 
         assertFalse(group.contains(notInAux));
     }
+
+    @Test
+    public void ReturnsTrueIfPathRelative() throws Exception {
+        Path auxFile = Paths.get(TexGroupTest.class.getResource("paper.aux").toURI());
+        String user = System.getProperty("user.name") + '-' + InetAddress.getLocalHost().getHostName();
+        metaData.setLaTexFileDirectory(user, auxFile.getParent().toString());
+        TexGroup group = new TexGroup("paper", GroupHierarchyType.INDEPENDENT, auxFile, new DefaultAuxParser(new BibDatabase()), new DummyFileUpdateMonitor(), metaData);
+
+        assertTrue("paper.aux".matches(group.getFilePath().toString()));
```

Comment:
Please use `assertEquals("paper.aux", group.getFilePath().toString())`. It gives a better error message in case the test fails.